clang/docs/ReleaseNotes.rst

```diff
@@ -124,7 +124,7 @@ AST Dumping Potentially Breaking Changes
     ``__atomic_test_and_set(p, 0)``
 
 - Pretty-printing of templates with inherited (i.e. specified in a previous
-  redeclaration) default arguments has been fixed.
+  redeclaration) and template default arguments has been fixed.
```

Comment:
Not sure if this is clear and correct.

Reply:
I'd advise to add a separate entry to the change log, and say something to the effect you have fixed pretty printing of default arguments to template template parameters.

Reply:
Done.